backend/curriculum_tracking/tests/test_activity_log_entry_creators.py

```diff
@@ -23,6 +23,33 @@ def test_that_only_one_entry_created_for_one_review(self):
         creators.log_project_competence_review_done(review2)
         self.assertEqual(LogEntry.objects.count(), 2)
 
+    def test_two_entries_created_for_two_reviews_with_second_done_within_debounce_period_by_another_user(self):
+        review1 = RecruitProjectReviewFactory()
+        review2 = RecruitProjectReviewFactory(
+            status=review1.status,
+            timestamp=review1.timestamp,
+            comments=review1.comments,
+            recruit_project=review1.recruit_project,
+            reviewer_user=UserFactory()
+        )
+        creators.log_project_competence_review_done(review1)
+        creators.log_project_competence_review_done(review2)
+        self.assertEqual(LogEntry.objects.count(), 2)
+
+    def test_logging_the_same_entry_twice_only_produces_one_log_entry(self):
+        review1 = RecruitProjectReviewFactory()
+        creators.log_project_competence_review_done(review1)
+        creators.log_project_competence_review_done(review1)
+        self.assertEqual(LogEntry.objects.count(), 1)
+
+    def test_log_project_creator_is_invoked_for_a_project_review(self):
+        review1 = RecruitProjectReviewFactory()
+        try:
+            creators.log_topic_competence_review_done(review1)
+        except AttributeError as error:
```

Comment:
There should not be these errors. 

If an exception is meant to be raised then you should assert that it is raised. But if it is not meant to be raised then dont squash it. Errors need to be fixed, not silenced.

Reply:
I will assert that the correct exception is raised, I deliberately called the wrong creator to test that calling the wrong creator won't work.